src/compiler/emitter.ts

```diff
@@ -659,6 +656,7 @@ namespace ts {
         getAllAccessorDeclarations: notImplemented,
         getSymbolOfExternalModuleSpecifier: notImplemented,
         isBindingCapturedByNode: notImplemented,
+        getDeclarationStatementsForSourceFile: notImplemented,

```

Comment:
Handling emit concating when output of d.ts file doesn't change but referencing project with prepend = true, (so transformation of existing output from .tsbuildinfo, existing output, new output from referenced projects to new output (look for testrunner\unittests\tsbuild\outFile.ts)